src/amo/components/ShowMoreCard/index.js

```diff
@@ -23,6 +23,7 @@ type UIStateType = {|
 |};
 
 type Props = {|
+  childId: number | string | null,
```

Comment:
Could we enforce `string`?

Reply:
We could, but it's convenient for it to allow different values. Then the caller doesn't need to worry about conversion. This seems better to me.